src/main/java/com/gam/api/dto/magazine/response/MagazineSearchResponseDTO.java

```diff
@@ -3,18 +3,23 @@
 import com.gam.api.entity.Magazine;
 import lombok.Builder;
 
+
 @Builder
 public record MagazineSearchResponseDTO (
         String thumbNail,
         String title,
         String interviewPerson,
+        String magazineUrl,
         Long viewCount
 ){
+    static String baseUrl = "https://flourishing-cocada-40a6ce.netlify.app/magazine/";
+
     public static MagazineSearchResponseDTO of(Magazine magazine) {
         return MagazineSearchResponseDTO.builder()
                 .thumbNail(magazine.getThumbNail())
                 .title(magazine.getMagazineTitle())
                 .interviewPerson(magazine.getInterviewPerson())
+                .magazineUrl(baseUrl+magazine.getId())
```

Comment:
간격을 두는게 더 보기 좋아 보입니다!
```suggestion
                .magazineUrl(baseUrl + magazine.getId())
```